eng/tools/dependency-testing/index.js

```diff
@@ -191,18 +191,9 @@ async function insertMochaReporter(targetPackagePath, repoRoot, testFolder) {
 async function insertTsConfigJson(targetPackagePath, testFolder) {
   const testPath = path.join(targetPackagePath, testFolder);
   var tsConfigJson = await packageUtils.readFileJson("./templates/tsconfig.json");
-  var tsConfigTestsJson = await packageUtils.readFileJson("./templates/tsconfig.tests.json");
```

Comment:
I don't know enough about this file to know what's going on here. Do these changes impact other packages? If so, did you make sure those packages still work after these changes?

Reply:
Actually I had done that change in another PR that is already merged to master - rebased my changes with the master and the dependency-testing/index.js changes should no longer be in this PR @chradek